### DISCUSSION TITLE
Slow downloads

### DISCUSSION THREAD

**Discussion by nollinvoyd** [aclap-dev/video-downloadhelper#1380] [Help and Q&A]
### Your question

Fx 101.03, Win 10 pro

Is there a way to speed things up.  I often find that when the download goes directly to the browser, or to FDM for instance, instead of passing thru VDH, it's considerably faster.  

Also, is there a way to have a persistent interface while downloading, either a tab or popup, instead of having to go thru the toolbar button? Can't that be done in the same tab setting used for initiating the download? 

I have my browser toolbar on top, so every time I want to check progress, or something else, I not only have to scroll to the bottom of the menu, but I have to do the same thing later if I want to go there again.

### Addon & CoApp version and details

version: 9.1.0.41
channel: beta
build date: 2024-08-02
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:131.0) Gecko/20100101 Firefox/131.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\mr",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

People come in here every so often claiming VDH downloads something slow.  I do not observe this.  In general, it is serving web sites that throttle their service that cause slow downloads.  It is not your ISP, not your computer, not your browser, not VDH.  It is the site you are trying to download from that slows things down on purpose.  If you wish somebody else to corroborate your experience, you need to post some examples.  An example is not the name of a web site.  An example is the URL of a specific video on a web site.  I'm not scrounging around on some site I've never visited before looking to replicate your results.  Give me some URLs of specific videos that download slow for you & I'll see if they download slow for me.

I'm not sure I follow the rest of your comments.  The VDH menu can show activity in any tab.  There's a setting `Show all tabs` that allows you to monitor downloads in any tab no matter what browser window you have in foreground.  As for the order of what's on the VDH menu, you can affect that through the VDH settings `Sort by status` & `Reverse sorting`.  Experiment with those & see if you can make things more to your taste.

You're right, you have to open the VDH menu to monitor status.  That's an interesting suggestion to have a persistent display somewhere.  You might want to separate that out to its own discussion so it can be considered properly.

**Reply by GrampaWildWilly** [reply]
You can also monitor activity in the Resource Monitor (the built-in Windows tool) on the Network tab.  That won't easily give you specific tracking information to connect that activity to a particular item you are downloading from a given web page, but it is persistent & it can give you some information.  I find it useful for determining just how fast or slow a download really is.  I intend to use it to see whether your claims of slow downloads are true for me.

**Comment by Mattmjs** [comment]
> is there a way to have a persistent interface while downloading

Yes of course with the sidebar, there is a button that says show in sidebar.
